### PR TITLE
[WIP] Add `test-result` command

### PR DESCRIPTION
**Pull request by shunjikonishi**

### CODE REVIEW

**Comment by shunjikonishi** [comment]
Add test-result command.

How to test with prod server

- OpenChallenge
  - codecheck test-result 2840
- PrivateExam
  - codecheck test-result 3071
  - codecheck test-result 3071 -u [USERNAME] -p [PASSWORD]

@takayukioda review me

File: src/commands/testResult.js [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+  console.log("Arguments:");
+  console.log("  ResultId: Challenge result id to run test.");
+  console.log("Options:");
+  console.log("  -u, --user    : username");
+  console.log("  -p, --password: password");
+};
+
+TestResultCommand.prototype.checkArgs = function(args) {
+  if (args.length === 0) {
+    throw "ChallengeId is required";
```
Comment (takayukioda) [inline src/commands/testResult.js]: Challenge Id? Isn't this a Result Id?

**Comment by takayukioda** [comment]
lgtm

File: src/commands/testResult.js (hunk 1 above)
Comment (shunjikonishi) [inline src/commands/testResult.js]: Oh, it is result id. I will fix before merge

**Pull request merged**